specification/columns/chargeclassification.md

```diff
@@ -0,0 +1,38 @@
+# Charge Classification

```

Comment:
It's almost like the purpose of the charge.

Reply:
We could move this to transaction type

Reply:
other option is transaction category